adapter/internal/adapter/adapter.go

```diff
@@ -223,6 +224,22 @@ func Run(conf *config.Config) {
 	if eventHubEnabled {
 		// Load subscription data
 		eventhub.LoadSubscriptionData(conf)
+		var isAzureEventingFeatureFlagEnabled bool
+		var err error
+
+		envValue := os.Getenv("FEATURE_FLAG_REPLACE_EVENT_HUB")
+		if (envValue != "") {
+			isAzureEventingFeatureFlagEnabled, err = strconv.ParseBool(envValue)
```

Comment:
Please add TODOs with your github  handle(user name) to remove these flag related code when feature is completed.